psp/scripts/eval_model.py

```diff
@@ -58,8 +57,7 @@ def main(exp_root, exp_name, exp_config_name, num_workers, limit):
 
     # Load the saved model.
     model_path = exp_root / exp_name / "model.pkl"
-    with open(model_path, "rb") as f:
-        model: PvSiteModel = pickle.load(f)
+    model = load_model(model_path)
```

Comment:
nice

Reply:
It's still pickle under the hood... with some extra hacks :P